apps/project/serializers.py

```diff
@@ -275,7 +275,7 @@ def create(self, validated_data: dict[str, typing.Any]) -> ProjectAsset:
         return super().create(validated_data)
 
 
-class OrganizationSerializer(UserResourceSerializer[Organization]):
+class OrganizationSerializer(UserResourceSerializer[Organization], ArchivableResourceSerializer[Organization]):
     class Meta:  # type: ignore[reportIncompatibleVariableOverride]
         model = Organization
-        fields = ("name",)
+        fields = ("name", "description", "is_archived")
```

Comment:
We don't need to add "is_archived"
It should be handled in ArchivableResourceSerializer